jicofo-selector/src/main/java/org/jitsi/jicofo/bridge/JvbDoctor.java

```diff
@@ -174,7 +174,7 @@ protected void doHealthCheck()
 
             logger.debug("Sending health-check request to: " + bridge);
 
-            IQ response = UtilKt.sendIqAndGetResponse(connection, newHealthCheckIQ(bridge));
+            IQ response = org.jitsi.jicofo.xmpp.UtilKt.sendIqAndGetResponse(connection, newHealthCheckIQ(bridge));
```

Comment:
Why does this need to be spelled out?  Do we have multiple `UtilKt` classes floating around now?

Reply:
Yes, we have one in jicofo-common and one in jicofo-selector